### ISSUE TITLE
The networking feature causes a lot of issues.

### ISSUE THREAD

**Issue by Lunxara** [remiX-/QualityCompany#20]
Okay so the Networking feature this mod has has given me constant issues sadly. :( If I leave it enabled then it'll conflict with loaforcsSoundAPI and some sounds won't work, If I disable it it fixes the conflict with the SoundAPI but then creates a conflict where it partially breaks the Networking feature of GhostCodes. Now I really love this mod specifically for the Hotbar features and I almost feel like the Hotbar features should be seperated into a seperate mod at this point just to avoid compat issues. But I hope you can maybe look into this and find out why the Networking stuff in this mod conflicts with other mods ^^

**Comment by zoqin** [comment]
I agree if its too hard to make the mod compatible as a whole, making a dedicated mod for the hotbar features would be greatly appreciated

**Comment by remiX-** [comment]
Hey @Lunxara @zoqin,

This is interesting :O We play with 68 installed mods at the moment including Lethal Resonance which has `loaforc-loaforcsSoundAPI` has a dependency so we have that as well and haven't noticed any issues.

Do you have a modpack I can try out and an example scenario I can play out to see whats happening?

In terms of the UI - yep it seems it's definitely the favourite and it was mentioned in another issue to separate it as well :D

**Comment by zoqin** [comment]
Personally I just agree on the idea of making a HUD dedicated mod if its too much trouble for the whole to be compatible personally the most annoying conflict I seen is the #15 , as its game breaking I didn't experiment more 

So I can't provide much information for loaforcsSoundAPI sorry but i will happily provide more for 15# if you want

**Comment by Lunxara** [comment]
> Hey @Lunxara @zoqin,
> 
> This is interesting :O We play with 68 installed mods at the moment including Lethal Resonance which has `loaforc-loaforcsSoundAPI` as a dependency so we have that as well and haven't noticed any issues.
> 
> Do you have a modpack I can try out and an example scenario I can play out to see whats happening?
> 
> In terms of the UI - yep it seems it's the favourite and it was mentioned in another issue to separate it as well :D
> 
> Does disabling the networking via the config solve it at all?

It won't be entirely apparent, but when you disable the networking feature you'll notice some sounds from Lethal Resonance will work that didn't before, such as the Spore Lizard sounds as well as some other sounds like the Coil Head Sounds. It's really strange that it even conflicts to begin with tbh
